app/controllers/groups_controller.rb

```diff
@@ -3,7 +3,7 @@ class GroupsController < ApplicationController
 
   before_action :authenticate_user!, except: [ :show ]
   before_action :find_group, except: [:index, :new, :create, :feature]
-  before_action :is_group_admin, only: [:update, :destroy, :update]
```

Comment:
Why was `update` removed from is_group_admin? Where is the test case for this? This implies that a group_member can update the group?